app/src/main/java/zafrani/com/pubgapp/models/Type.java

```diff
@@ -0,0 +1,37 @@
+package zafrani.com.pubgapp.models;
+
+import android.support.annotation.NonNull;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.List;
+
+public class Type {
+    @NonNull
+    @SerializedName("type_name")
+    private final String type_name;
+    @NonNull
+    @SerializedName("items")
+    private final List<Item> items;
+
+    public Type(@NonNull String type_name,
```

Comment:
This should just be name. Its obviously a type name since its inside the type object